ui-modules/blueprint-composer/app/components/spec-editor/spec-editor.template.html

```diff
@@ -688,9 +694,9 @@ <h4 ng-if="model.hasEnrichers() && filteredEnrichers.length === 0">No enrichers
 
 <ng-include src="'blueprint-composer/component/spec-editor/section-others.html'"></ng-include>
 <script type="text/ng-template" id="blueprint-composer/component/spec-editor/section-others.html" defer-to-preexisting-id="true">
+    <!-- Placeholder for other sections. -->
 </script>
```

Comment:
I would re-use 'others' section for requirements, however, it is not in the desired position. We agreed to have requirements displayed right under configuration.